src/screens/ChatScreen/components/ChatHeader.js

```diff
@@ -106,6 +155,31 @@ const ChatHeader = ({
         }),
       );
     }
+    if (itemType === 'mute_conversation') {
+      const { muted } = conversationDetails;
+      if (!muted) {
+        dispatch(muteConversation({ conversationId }));
+      }
+    }
+    if (itemType === 'unmute_conversation') {
+      const { muted } = conversationDetails;
+      if (muted) {
+        dispatch(unmuteConversation({ conversationId }));
+      }
+    }
+  };
+
+  const toggleStatusForConversations = () => {
+    setLoadingStatus(true);
+    try {
+      captureEvent({ eventName: 'Toggle conversation status' });
+      dispatch(toggleConversationStatus({ conversationId }));
+    } catch (error) {
+    } finally {
+      setTimeout(() => {
+        setLoadingStatus(false);
+      }, 1000);
```

Comment:
This seems like a hack, @muhsin-k Can you please guide @iamsivin on using the uiFlags in the store itself instead of doing it here?

Reply:
Done https://github.com/chatwoot/chatwoot-mobile-app/pull/522/commits/2806ba0b3fd6d0873346bcdcdb57871be090393f